web/styles/compose.css

```diff
@@ -188,19 +188,49 @@
         }
 
         & a.narrow_to_compose_recipients {
-            background: transparent;
-            font-size: 18px;
-            padding: 0 1px;
-            align-self: center;
+            display: flex;
+            align-items: center;
+            background: hsl(227deg 78% 59% / 12%);
+            color: hsl(227deg 78% 59% / 100%);
+            font-size: 20px;
+            padding: 0 5px;
+            height: 100%;
             line-height: 20px;
             opacity: 0.7;
-            border: 0;
-            margin-left: 3px;
             text-decoration: none;
-            color: inherit;
+            transition: all 100ms ease-in-out;
+            border-radius: 3px 0 0 3px;
+            /* Positioned absolutely so it takes up full height but
+            stays inside its flex parent when its other children (pills)
+            wrap onto another row */
+            position: absolute;
```

Comment:
I kind of don't love this use of relative/absolute positioning. I don't know whether it's worth trying to get it into this PR, but it seems to me like the the container ought to be defined as a grid, with one column for the icon and the other for the pills (which might need their own flex container).

Reply:
And also, a gridded presentation like this is probably going to be useful for pulling off the overlap-style presentation in the demo.

Reply:
I've made a prep commit giving the container a grid layout! 

Also, I realised that the transparency of the element overlapping the new button was causing the button to show through, so I changed those transparent colors to solid ones, but they do have the same look as before. @karlstolley Please let me know if that makes sense. Have elaborated a little more in comments in the css.

@alya Now that the slight overlapping works, and things look near identical to the demo, you might want to give this PR another test, or see the updated screenshots in the PR description.